rs/sns/governance/src/pb/conversions.rs

```diff
@@ -79,14 +79,21 @@ impl From<pb_api::DisburseMaturityInProgress> for pb::DisburseMaturityInProgress
 
 impl From<pb::Neuron> for pb_api::Neuron {
     fn from(item: pb::Neuron) -> Self {
-        let topic_followees = Some(
-            item.topic_followees
-                .into_iter()
-                .map(|(topic, followees)| {
-                    let followees = pb_api::neuron::Followees::from(followees);
-                    (topic, followees)
-                })
-                .collect(),
+        let topic_followees = item.topic_followees.map(
+            |pb::neuron::TopicFollowees {
+                 topic_id_to_followees,
+             }| {
+                let topic_id_to_followees = topic_id_to_followees
+                    .into_iter()
+                    .map(|(topic, followees)| {
+                        let followees = pb_api::neuron::Followees::from(followees);
+                        (topic, followees)
+                    })
+                    .collect();
+                pb_api::neuron::TopicFollowees {
```

Comment:
```suggestion

                pb_api::neuron::TopicFollowees {
```

Reply:
It doesn't seem that we're close to being consistent w.r.t. newlines in this particular source file. 

I thus suggest reformatting it altogether in a separate PR.